texttospeech/test/quickstart.test.mjs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs/promises';
+import { existsSync } from 'node:fs';
+import { describe, it } from 'mocha';
+import * as cp from 'child_process';
+import { assert } from 'chai';
+
+const outputFile = 'quickstart_output.mp3';
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+describe('quickstart', () => {
+    before(() => {
+        try {
+            fs.unlinkSync(outputFile);
+        } catch (err) {
+            // Ignores error if unable to delete the outputFile
+        }
+    })
+
+    it('should synthesize speech to local mp3 file', async () => {
+      // Verifies that outputFile doesn't exist 
+      assert.strictEqual(existsSync(outputFile), false);
```

Comment:
suggestion: Since this first statement is checking there's no pre-existing file that might confuse test results or be overwritten, it would be helpful to use the third parameter in this assert to provide instructions:

```suggestion
      assert.strictEqual(existsSync(outputFile), false, `found pre-existing ${outputFile}, please rename or remove and retry the test`);
```